Tests/Grid/Mapping/SourceTest.php

```diff
@@ -0,0 +1,240 @@
+<?php
+
+namespace APY\DataGridBundle\Grid\Tests\Mapping;
+
+use APY\DataGridBundle\Grid\Mapping\Source;
+use PHPUnit\Framework\TestCase;
+
+class SourceTest extends TestCase
+{
+    public function setUp()
+    {
+        $this->source = new Source([]);
+    }
+
+    public function testSetterColumnsWithDefaultValue()
+    {
+        $this->assertAttributeEquals(
+            [],
+            'columns',
+            $this->source
+        );
+    }
+
+    public function testSetterFilterableWithDefaultValue()
+    {
+        $this->assertAttributeEquals(
+            true,
+            'filterable',
+            $this->source
+        );
+    }
+
+    public function testSetterSortableWithDefaultValue()
+    {
+        $this->assertAttributeEquals(
+            true,
+            'sortable',
+            $this->source
+        );
+    }
+
+    public function testSetterGroupsWithDefaultValue()
+    {
+        $expectedGroups = array('0' => 'default');
+
+        $this->assertAttributeEquals(
+            $expectedGroups,
+            'groups',
+            $this->source
+        );
+    }
+
+    public function testSetterGroupByWithDefaultValue()
+    {
+        $this->assertAttributeEquals(
+            [],
+            'groupBy',
+            $this->source
+        );
+    }
+
+    public function testGetterColumnWithDefaultValue()
```

Comment:
Those `getter`s with default values are useless as you're already testing attributes. They are meaningful after you set something in them and check for getter return value, so you can remove them as you're already testing for correct getter values below.